### PR TITLE
Function/8 string vowel extract.java

### PR DESCRIPTION
**Pull request by ShubhamNakashe**
#What does this PR do ?

#8 Find Vowels from a Word/String 

So in src/main/java 
I made a package vowelExtract
It contains 2 java files 
one is for the testing 
second is for stringVowelExtracting

image
image

### CODE REVIEW

File: src/main/java/vowelExtract/stringVowel.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package vowelExtract;
+import java.util.Scanner;  // Import statement for Scanner
+
+public class stringVowel {
+
```
Comment (srinivasboga) [inline src/main/java/vowelExtract/stringVowel.java]: Java convention is to capitalize class names `StringVowel `

File: src/main/java/vowelExtract/stringVowel.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            if ("aeiouAEIOU".indexOf(a) != -1) {
+                s.append(a);
+            }
+        }
+        return s.toString();
+    }
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String str = sc.nextLine();
```
Comment (srinivasboga) [inline src/main/java/vowelExtract/stringVowel.java]: 1. For console applications, use Scanner for input. 
2. Alternatively, consider using javax.swing.JOptionPane.showInputDialog() for GUI-based input.

**Review by srinivasboga — CHANGES_REQUESTED** [review]

File: src/main/java/vowelExtract/stringVowel.java (hunk 1 above)
Comment (ShubhamNakashe) [reply src/main/java/vowelExtract/stringVowel.java]: ok I will change this

File: src/main/java/vowelExtract/stringVowel.java (hunk 2 above)
Comment (ShubhamNakashe) [reply src/main/java/vowelExtract/stringVowel.java]: So I must use javax.swing.JOptionPane.showInputDialog() in this case right ??

Comment (ShubhamNakashe) [reply src/main/java/vowelExtract/stringVowel.java]: Also can you check the code for test as I have written it for the first time

**Comment by srinivasboga** [comment]
@ShubhamNakashe 

there is an extra unwanted lines of code is there remove those,
in file : src/main/java/DateAndTime/LocalDateExample.java

`
 //write new date operations
		DateOperations dateOperations = new DateOperations();
		dateOperations.dateOperations(localDate);
		System.out.println(localDate);

		//write to find out if its a leap year

		LeapYear leapYear = new LeapYear();
		leapYear.leapYear(localDate);
`

or else give me the branch name I'll fork it and do the changes.

**Comment by ShubhamNakashe** [comment]
Ok I will remove those lines of code. Is there any other issue with the code that I have uploaded or is it fine ?? 
Any changes needed ??

**Comment by ShubhamNakashe** [comment]
Hello @srinivasboga 
I checked the build file errors and tried to fix the error that was due to the above lines of codes and secondly another error caused due to javax.swing occurred due to java.desktop 
I also fixed that error So Shall I send my PR request now ??

**Comment by ShubhamNakashe** [comment]
Hello @srinivasboga ??

**Comment by srinivasboga** [comment]
> Hello @srinivasboga ??

Yes @ShubhamNakashe, send ur pr branch_name

**Comment by srinivasboga** [comment]
@ShubhamNakashe 

You can see in the image that it indicates a file name change. You changed the class name directly in the editor, but you need to use the 'Rename file' option in your tool. Only then will the original file be updated.
image

**Comment by ShubhamNakashe** [comment]
Hello @srinivasboga I have refactored the class name and committed the changes can you pls check ??

**Pull request merged**